patches/chrome-android-BUILD.gn.patch

```diff
@@ -43,7 +43,14 @@ index f0c7c5c7f7ebb088375e41c2292071cb734d823b..af14f2c5fa4ee6c466337efa2fec498e
  }
  
  android_library("bundle_canary_java") {
-@@ -1410,6 +1414,7 @@ if (current_toolchain == default_toolchain) {
+@@ -1404,12 +1408,14 @@ if (current_toolchain == default_toolchain) {
+         "$target_gen_dir/${_variant}_paks/chrome_100_percent.pak",
+         "$target_gen_dir/${_variant}_paks/resources.pak",
+       ]
++      sources += brave_android_assets
```

Comment:
we already have `brave_pak_assets` for this

Reply:
good catch, fixed it